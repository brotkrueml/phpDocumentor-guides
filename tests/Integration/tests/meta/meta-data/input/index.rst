:nosearch:
:orphan:
:nocomment:
:project:
    My Project
:abstract:
    Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt
    ut labore et dolore magna aliquyam erat, sed diam voluptua.
:author:
    Contributors
:contact:
    https://www.phpdoc.org/
:copyright:
    MIT
:dedication:
    At vero eos et accusam et justo duo dolores et ea rebum.
:organization:
    phpDocumentor
:revision:
    Some Revision
:version:
    3.1.4

==============
Document Title
==============

Lorem Ipsum Dolor.
