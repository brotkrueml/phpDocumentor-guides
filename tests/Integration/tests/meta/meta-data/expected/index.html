<!DOCTYPE html>
<html lang="en">
<head>
    <title>Document Title - My Project</title>
<meta name="robots" content="noindex">

    <meta name="description" content="Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt
ut labore et dolore magna aliquyam erat, sed diam voluptua." />
<meta name="author" content="Contributors" />

<meta name="copyright" content="MIT" />



</head>
<body>
    <div class="section" id="document-title">
    <h1>Document Title</h1>

            <p>Lorem Ipsum Dolor.</p>
    </div>

</body>
</html>
